acme/http_challenge.go

```diff
@@ -95,12 +95,13 @@ Loop:
 func (s *httpChallenge) startHTTPServer(domain string, token string, keyAuth string) {
 
 	// Allow for CLI port override
-	port := ":80"
+	port := domain + ":80"
```

Comment:
It wasn't using that before (on L103). I can rename it to 'listen', that makes sense.